doc/install.md

```diff
@@ -1,4 +1,9 @@
-## Installation
+# Installation
+
+
+## From binaries:
+- Wheels for Mac and Linux can be installed via `pip install cesium`
+- We do not build binary wheels for Windows. To install on Windows, follow the instructions below for installation from source.
```

Comment:
I did publish 64-bit Windows wheels for the last release. I cannot confirm whether they work or not.

Either way, I think this section should be removed.